Dockerfile

```diff
@@ -18,5 +29,7 @@ COPY --from=build /build/package*.json ./
 COPY --from=build /build/.next ./.next
 COPY --from=build /build/public ./public
 RUN npm install next
+RUN echo $BUILD_DATE 
+RUN echo $TC_BUILD
```

Comment:
We could remove this, right? All the same, doesn't hurt to keep it.

Reply:
gonna keep em in for now (helps for debugging, but could remove them in the future for sure).